src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -43,44 +25,199 @@
+                    "longname",
+                    "profile_type",
+                    "units"
+                  ],
+                  "type": "object"
+                }
+              },
+              "type": "object"
+            }
+          },
+          "type": "object"
+        },
+        "files_to_copy": {
+          "$ref": "#/$defs/filesToStage"
+        },
+        "files_to_link": {
+          "$ref": "#/$defs/filesToStage"
         },
         "length": {
           "minimum": 1,
           "type": "integer"
         },
         "model_configure": {
-          "$ref": "#/$defs/updatable_config"
-        },
-        "mpicmd": {
-          "type": "string"
+          "additionalProperties": false,
+          "properties": {
+            "base_file": {
+              "type": "string"
+            },
+            "update_values": {
+              "minProperties": 1,
+              "patternProperties": {
+                "^.*$": {
+                  "type": [
+                    "boolean",
+                    "number",
+                    "string"
+                  ]
+                }
+              },
+              "type": "object"
+            }
+          },
+          "required": [
+            "base_file"
+          ],
+          "type": "object"
         },
         "namelist": {
-          "$ref": "#/$defs/updatable_config"
+          "additionalProperties": false,
+          "properties": {
+            "base_file": {
+              "type": "string"
+            },
+            "update_values": {
+              "minProperties": 1,
+              "patternProperties": {
+                "^.*$": {
+                  "minProperties": 1,
+                  "patternProperties": {
+                    "^.*$": {
+                      "minProperties": 1,
+                      "type": [
+                        "array",
+                        "boolean",
+                        "number",
+                        "string"
+                      ]
+                    },
+                    "type": "object"
+                  },
+                  "type": "object"
+                }
+              },
+              "type": "object"
+            }
+          },
+          "required": [
+            "base_file"
+          ],
+          "type": "object"
         },
         "run_dir": {
-          "format": "uri",
           "type": "string"
         },
-        "ufs_configure": {
-          "format": "uri",
-          "type": "string"
+        "runtime_info": {
+          "additionalProperties": false,
+          "properties": {
+            "mpi_args": {
+              "items": {
+                "type": "string"
+              },
+              "type": "array"
+            },
+            "threads": {
+              "type": "integer"
+            }
+          },
+          "type": "object"
         }
       },
+      "required": [
+        "field_table",
+        "run_dir",
+        "runtime_info"
+      ],
```

Comment:
@christinaholtNOAA I suspect that this list is incomplete. Aren't `domain`, `executable`, `length`, `namelist`, possible others also actually required? Although their contents might be variable, isn't at least _one_ file to link and one file to copy required for a run, such that the (new) `files_to_copy` and `files_to_link` blocks would also be required?

Reply:
After our discussion yesterday, I think it might make sense for us to handle only those entries that are not file-related here in the schema: `domain`, `executable`, `length`, `run_dir`, `runtime_info`.

All others, I think we can leave up to the user whether they'd prefer to have them staged or handled via a config entry for either template or config handling. The task would enforce their existence before run time.